src/data/sql/brp/create.gezagsverhoudingen.sql

```diff
@@ -0,0 +1,88 @@
+CREATE TABLE brp_prep.gezagsverhoudingen AS
+
+  SELECT
+    gv."BSN"::varchar                                                                AS burgerservicenummer,
+    gv."Anummer"::varchar                                                            AS anummer,
+    gv."IndGezagMinderjarigeOms"::varchar                                            AS indicatie_gezag_minderjarige,
+    gv."IndCurateleRegisterOms"                                                      AS indicatie_curatele_register, -- TODO: Check this
+    gv."GemeenteOntleningCode"::varchar                                              AS gemeente_gezagsverhouding_document,
+    CASE -- datum ontlening gezagsverhouding
+      WHEN gv."DatumOntlening" IS NULL THEN NULL
+      WHEN gv."DatumOntlening" = '0' THEN NULL
+      ELSE JSONB_BUILD_OBJECT(
```

Comment:
This piece of SQL is repeated so often that it's become hard to read and prone to bugs.

I think we should create a function (brp_build_date_json) as part of this pipeline that extracts this logic. Maybe even include the complete CASE expression as well from line 9. This also makes it much easier later on when we got a definitive answer about how to handle dates.